lib/mayaUsd/utils/stageCache.h

```diff
@@ -34,7 +34,7 @@ class UsdMayaStageCache
     /// 2 stage caches are maintained; 1 for stages that have been
     /// force-populated, and 1 for stages that have not been force-populated.
```

Comment:
Would be good to update the docstring here. Maybe just move the comments added in the cpp into the docstring here instead so that they get included in the generated documentation?